changelog/_unreleased/2020-09-18-token-change-event.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Dispatch event after token has been changed
+issue: NEXT-10941
+author: Kevin Chen
+author_email: kevin.chen@perfecthair.ch
+author_github: @maqavelli
+---
+# Core
+*  Add new event 'SalesChannelContextTokenChangedEvent' and dispatch in SalesChannelContextPersister replace method
```

Comment:
use `Added` as keyword

use backticks `` for classes

use full namespace

Reply:
@mitelg it's done :)